main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -291,7 +291,8 @@ function registerImporting() {
   IM.registerMimeType("application/x-excel", "binary/text/xml/xls/xlsx");
   IM.registerMimeType("application/xls", "binary/text/xml/xls/xlsx");
   IM.registerMimeType("application/x-xls", "binary/text/xml/xls/xlsx");
-  
+  IM.registerMimeType("application/vnd.openxmlformats-officedocument.spreadsheetml.*", "binary/text/xml/xls/xlsx");
```

Comment:
* I'm not sure if wildcards work here
* application/vnd.ms-excel already has an entry

Reply:
good catch, wildcards are indeed not supported yet, sadly. Perhaps it is still worth keeping here for reference, it does not hurt: I'll add an issue to add support for wildcards which would enable it. Unless there is a small number of possible values for this wildcard, in which case we could add all the variants directly?

Reply:
If we keep this line for now, we should add a comment about the fact that the wildcard is not supported yet: https://github.com/OpenRefine/OpenRefine/issues/2598